utilities/pipelines/sharedScripts/Get-ModuleTestFileList.ps1

```diff
@@ -35,7 +35,7 @@ function Get-ModuleTestFileList {
 
     $deploymentTests = @()
     if (Test-Path (Join-Path $ModulePath $SearchFolder)) {
-        $deploymentTests += (Get-ChildItem -Path (Join-Path $ModulePath $SearchFolder) -Depth 0 -Include ('*.json', '*.bicep') -File).FullName
+        $deploymentTests += (Get-ChildItem -Path (Join-Path $ModulePath $SearchFolder) -Recurse -Include ('*.json', 'deploy.test.bicep') -File).FullName
```

Comment:
Why not make `('*.json', 'deploy.test.bicep')` as an array parameter with default values?

Reply:
We could. Just happens that we use this implementation in different locations. Will update it for this script :)

Reply:
done